.gitignore

```diff
@@ -1 +1,4 @@
 config.py
+__pycache__/
+tutorial-env
+.vscode
```

Comment:
Can you add a new line at the end of this file? You can configure your VS code to automatically insert a new line whenever you save the file: https://stackoverflow.com/questions/44704968/visual-studio-code-insert-new-line-at-the-end-of-files

Ref: https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file